src/Component/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styled from "styled-components";
+import { AvatarCompSize, AvatarImgSize } from "../../variables";
+
+export interface ImgProps {
+    imgsrc?: string;
+    order: number
+};
+
+const ImgComp = styled.div<ImgProps>`
+height:${AvatarCompSize.max_height};
+width:${AvatarCompSize.max_width};
+border-radius:50%;
+display:flex;
+align-item:center;
+order:${props=>props.order};
+`;
```

Comment:
You have added ```margin: auto``` in ```img``` tag to align it in the center you can also add ```justify-content: center``` here it will do the same.

P.S. Not sure which is the correct way you can share your thoughts